### ISSUE TITLE
something wrong with scope

### ISSUE THREAD

**Issue by esclkm** [brantwills/Angular-Paging#51]
I confugured your paging - and its working super! thanks
but when i insert it inside bootstrap ui (0/12/1) tabset  -it lossing all config in the scope variable
![image](https://cloud.githubusercontent.com/assets/780145/15152594/7fb4ace2-16de-11e6-8a9a-9b632af03628.png)
![image](https://cloud.githubusercontent.com/assets/780145/15152676/d4af7858-16de-11e6-8c0c-19ec92231096.png)

how I can improve it?

**Comment by brantwills** [comment]
Hey esclkm, 

I've created a plunker to help explore this issue - feel free to modify to better suite your environment
http://plnkr.co/edit/LjuV0L0J40n5TzRJ0d7t?p=preview

**It looks like we must use angular >= v1.3.1 for the paging directive**
v1.3.1 introduced support in `$collection` for `$watchCollection` to be used in bi-directional bindings
https://github.com/angular/angular.js/blob/master/CHANGELOG.md#features-35

It seems without bi-directional bindings ONLY the attributes we manually declare are considered
I found that by manually declaring attributes I was able to get this to work with angular 1.2.
For example, below and in the plunker I am manually setting the `ul-class` attribute to `pagination`

```html
<div paging 
  ul-class="pagination"
  page="curPage" 
  page-size="pageSize" 
  total="totalItems">
</div>
```

I need to think about if we should update the dependencies to angular v1.3.1 in the node and bower configs OR if we should drop the `$watchCollection` and just go back to using three `$watch` statements to support older angular versions.  

I will aim to have this wrapped up by the weekend. I'd like to dig around a bit more to make sure there are no other older angular 1.2.x issues we may encounter.  

Thank you so much for bringing this issue up!
